draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+
+\subsection*{\MakeUppercase{AI Hero Setup}}
+
+Make AI Card Deck for Settlement defenders.
+
+\textbf{Settlement defenders Deck:} 1×Attack Card, 1×Defence Card, 1×Spell Card
+
+\textbf{Settlement defenders Spell Deck:} 1×Magic Arrow
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Conquer all Settlements by any player.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+\begin{itemize}
+  \item Any player lose his town.
+  \item All settlements are not owned by any player at the end of 7th turn.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+\begin{itemize}
+  \item At any time, Players can spend their Building Token to perform the same actions as if they visited a Trading Post.
+  \item When defending the town, and your Hero is not here, you can still use your Units (for free).
+  \item Each settlement is defend by clans Settlement defenders. (see below)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+  \begin{itemize}
+   \item \textbf{\nth{1} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.
+    \end{itemize}
+   \item \textbf{\nth{2} Round:}
+    \begin{itemize}
+      \item  First wave of enemy clan armies attacks the towns. (see below)
+      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
+    \end{itemize}
+   \item \textbf{\nth{3} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
+    \end{itemize}
+   \item \textbf{\nth{4} Round:}
+    \begin{itemize}
+      \item Second wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+     \end{itemize}
+   \item \textbf{\nth{6} Round:}
+    \begin{itemize}
+      \item Third wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+    \end{itemize}
+   \item \textbf{\nth{7} Round:}
+    \begin{itemize}
+      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
+    \end{itemize}
+  \end{itemize}
+
+\subsection*{\MakeUppercase{Enemy's clans waves}}
+\begin{itemize}
+  \item If enemy's settlement is not conquered, then enemy clans armies are attacking your castle!
+  \item At the start of the each player's turn, the battle begins.
+  \item Before attack, you can use your building, spell book and population tokens, and use Might and Magic Cards, but you can't move with Hero.
```

Comment:
```suggestion
  \item Before combat, you may use your Building, Spell Book, and Population Tokens, as well as Might and Magic Cards. However, you cannot move your Hero.
```
What do you mean by using Might and Magic Cards here?

Reply:
For example you can use  Legion arm artefact for decrease unit cost. Or play gold sack for money. (cuz at start you will draw new cards)

Reply:
So basically, use instant cards? If that's the intention, then this sentence is redundant, because you can always use them outside combat. Or maybe mention instant cards explicitly?